plugins/modules/cloud/packet/packet_device.py

```diff
@@ -424,6 +431,20 @@ def get_device_id_list(module):
     return device_ids
 
 
+def get_tags_list(module):
+    tags = module.params.get('tags')
+
+    if isinstance(tags, str):
+        tags = listify_string_name_or_id(tags)
+
+    tags = [dt.strip() for dt in tags]
+
+    if len(tags) > MAX_DEVICES:
```

Comment:
Not sure if this check is required, I suppose the Packet API manages this field as a string

Reply:
Yes on Packet they are a list of strings. I am not sure if there is a Limit, but I can check

Reply:
I agree. Removed the check.